### ISSUE TITLE
iOS PWA sometimes cannot go back

### ISSUE THREAD

**Issue by gagegogan** [actualbudget/actual#2152]
### Verified issue does not already exist?

- [X] I have searched and found no existing issue

### What happened?

Occasionally on iOS, if you install the app as a PWA. It will open you into the ad transaction screen and you are not able to go back with the back button.

https://github.com/actualbudget/actual/assets/10763068/1c064b82-e82c-4a05-b4e1-6dcc909aa363

### What error did you receive?

None

### Where are you hosting Actual?

Docker

### What browsers are you seeing the problem on?

Chrome, Safari

### Operating System

Mobile Device

**Comment by joel-jeremy** [comment]
Was the add transaction page the first page?

**Comment by gagegogan** [comment]
Yes

**Comment by gagegogan** [comment]
I'm not familiar with the code base. But I presume it has something to do with the history of the browser.

**Comment by gagegogan** [comment]
It definitely has to do with history. Although, I believe there's a web manifest argument to provide a start URL. That could be another work around. Let me try and re-create this issue a few more times and see exactly when it happens.

**Comment by jsehnoutka** [comment]
I have the same issue on Android, in case Actual is published as homescreen shortcut. Using the latest Edge.

**Comment by gagegogan** [comment]
Yep that's when I have it. It can be fixed by either changing the way history works or fixing the root cause that's opening the add transaction screen whenever you open the app. The way around it is to tilt to landscape and then back to portrait mode.

**Comment by jsehnoutka** [comment]
Thanks for the tip! I used to bypass it by using the Mobile version in browser, but this is also good workaround. 

It's weird though, I am prety sure this is a fresh issue. I update my Edge image frequently, but this issue started showing itself very recently (days at most), at least on my device.

**Comment by Kidglove57** [comment]
Will definitely look out for this - I'm on ioS Safari and have not seen it yet despite using the web responsive mobile version daily. I'm on Fly.

**Comment by gagegogan** [comment]
@Kidglove57 it needs to be in pwa mode

**Comment by Kidglove57** [comment]
Yes, always in PWA on my iPhone. Will watch out for this though. Mine always defaults to the Budget screen on opening.

**Comment by gagegogan** [comment]
I have chrome so that might affect it. Mine opens as a standalone so I'm not sure witch browser

**Comment by Kidglove57** [comment]
I just have the Safari link saved to the home screen and it then opens as an app. I may try Chrome.

**Comment by matt-fidd** [comment]
I believe this has now been fixed, I'll close it off but feel free to reopen if you can still reproduce.
